src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -45,12 +43,21 @@ public function __construct(string $messageBusService = 'message_bus', string $m
      */
     public function process(ContainerBuilder $container)
     {
-        if (!$container->hasDefinition($this->messageBusService)) {
+        if (!$container->hasDefinition($this->messageHandlerResolverService)) {
             return;
         }
 
-        if (!$container->getParameter('kernel.debug') || !$container->hasAlias('logger')) {
-            $container->removeDefinition('messenger.middleware.debug.logging');
+        $collectorDefinition = $container->getDefinition('messenger.data_collector');
+        foreach ($container->findTaggedServiceIds('messenger.bus') as $busId => $tags) {
```

Comment:
for consistency maybe `$this->busTag`, or at this point _just_ inline magic strings =/

no real opinion :)

Reply:
I changed it to be consistent and only inject the tag names as the configuration; it makes more sense.